.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "bergamot-translator-tests"]
 	path = bergamot-translator-tests
 	url = https://github.com/browsermt/bergamot-translator-tests
+[submodule "3rd_party/L4"]
+	path = 3rd_party/L4
+	url = https://github.com/microsoft/L4
```

Comment:
I'm very confused here, where is your fork located?

Reply:
https://github.com/browsermt/L4/pull/1. This probably happened because I started with a Microsoft submodule, opened a branch but eventually pushed to browsermt/L4. GitHub is able to lookup by commit across forks, so no fail.

Reply:
(I will fix this).